addons/l10n_ua/data/account_tax_template.xml

```diff
@@ -0,0 +1,485 @@
+            <field name="type_tax_use">purchase</field>
+            <field name="account_id" ref="ua_psbp_6441"/>
+            <field name="refund_account_id" ref="ua_psbp_6441"/>
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.tax_tag_vat_free')])]"/>
+        </record>
+        <record id="purchase_tax_template_vat_not_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">24</field>
+            <field name="name">Придбання Не є об'єктом ПДВ</field>
+            <field name="description">Не є об'єктом ПДВ</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">purchase</field>
+            <field name="account_id" ref="ua_psbp_6441"/>
+            <field name="refund_account_id" ref="ua_psbp_6441"/>
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.tax_tag_vat_not')])]"/>
+        </record>
+        <!-- Simplified tax system -->
+        <!-- Sale taxes -->
+        <record id="simple_tax_sale_product_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">30</field>
+            <field name="name">Дохід від продажу  товарів</field>
+            <field name="description">Дохід від продажу товарів</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">sale</field>
+        </record>
+        <record id="simple_tax_sale_gift_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">31</field>
+            <field name="name">Дохід від безоплатно отриманих  товарів</field>
+            <field name="description">Дохід від безоплатно отриманих товарів</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">sale</field>
+        </record>
+        <record id="simple_tax_sale_old_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">32</field>
+            <field name="name">Дохід від заборгованності за якою минув строк позивної давності</field>
+            <field name="description">Дохід від заборгованності, за якою минув строк позивної давності</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">sale</field>
+        </record>
+        <record id="simple_tax_sale_15_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">33</field>
+            <field name="name">Дохід, за ставкою 15%</field>
+            <field name="description">Дохід за ставкою 15%</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">sale</field>
+        </record>
```

Comment:
Should not we restrict to just VAT?

Reply:
yes type "none" should be here

Reply:
We use it for sale. It is for simplified tax system (cash basis).